.github/workflows/tests.yml

```diff
@@ -0,0 +1,53 @@
+name: Tests
+
+on:
+  pull_request:
+    types: [opened, closed, synchronize]
+  workflow_dispatch:
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+
+          # Dependencies to build the deb
+          # sudo apt-get install build-essential devscripts debhelper -y
+
+          # Dependencies for deb-get itself
+          sudo apt-get install apt apt-transport-https bash coreutils curl dpkg grep libc-bin lsb-release sed software-properties-common sudo wget -y
+
+
+      # All these commands should return a 0 status
+      - name: Test zero status
+        run: |
+          ./deb-get list >/dev/null
+          ./deb-get help >/dev/null
+          ./deb-get cache >/dev/null
+          ./deb-get version >/dev/null
+
+      # Install a couple of packages to see if they still install
+      - name: Test package installations
+        run: |
+          export DEBIAN_FRONTEND=noninteractive
+          set -x
+          # @todo fix firefox-esr see https://github.com/OhMyMndy/deb-get/runs/7467951397?check_suite_focus=true
+          ./deb-get install firefox-esr || true
+          ./deb-get install 1password >/dev/null
+          ./deb-get install bitwarden >/dev/null
+          # @todo fix obs-studio see https://github.com/OhMyMndy/deb-get/runs/7467973508?check_suite_focus=true
+          ./deb-get install obs-studio  || true
```

Comment:
firefox-esr and obs-studio now work on upstream/main because there was a bug that is no more.
Thanks for the loan of your code and the awesome learning day I've had today.